app/controllers/events/view/sessions/create.js

```diff
@@ -6,6 +6,9 @@ export default Controller.extend({
       await this.get('model.session').save();
       if (this.addNewSpeaker) {
         let newSpeaker = this.get('model.speaker');
+        if (newSpeaker.isEmailOverridden) {
+          newSpeaker.set('email', this.authManager.currentUser.email);
+        }
```

Comment:
As this code is repetitive, is it possible to shift it to a mixin?

Reply:
I think that these 2 times are should not be considered for any mixin or helper. I think it's fine